html/elements/fieldset.json

```diff
@@ -58,11 +58,16 @@
                 "version_added": "20"
               },
               "chrome_android": "mirror",
-              "edge": {
-                "version_added": "12",
-                "partial_implementation": true,
-                "notes": "Does not work with nested fieldsets. For example: `&lt;fieldset disabled&gt;&lt;fieldset&gt;&lt;!--Still enabled--&gt;&lt;/fieldset&gt;&lt;/fieldset&gt;`"
-              },
+              "edge": [
+                {
+                  "version_added": "79"
+                },
```

Comment:
It would be handy if we could set `"mirror"` here, such that the pre-Chromium statement entry is merged with statements coming from Chrome.

/cc @Elchi3 @queengooborg